### DISCUSSION TITLE
Memory leak with React Native 0.74.2 and RN Gesture Handler 2.17.1

### DISCUSSION THREAD

**Discussion by satheshrgs-rksv** [software-mansion/react-native-gesture-handler#2959] [Q&A] [answered]
Hi

I am currently on React Native 0.71.1 and RN Gesture Handler 2.9.0.

I tried to update to React Native 0.74.2 and RN Gesture Handler 2.17.1. After upgrade facing memory leak. Memory usage keeps on increasing. I'm currently on a hybrid React Native App. Loading React Native View on a native app in a fragment (following [this](https://github.com/callstack/react-native-brownfield) brownfield architecture.

I tried to downgrade to version to 0.72 and there is not leak in that version. Facing leaks from version 0.73.x. Please help with this

**Comment by m-bert** [comment]
> I tried to downgrade to version to 0.72 and there is not leak in that version.

In that case are you sure that those leaks come from Gesture Handler? Because it looks like the problem lies in React Native.

**Reply by satheshrgs-rksv** [reply, marked as answer]
Yeah, seems so. I have raised an issue in react native repo - https://github.com/facebook/react-native/issues/45217

**Comment by m-bert** [comment]
> I have raised an issue in react native repo 

Given that I'm closing this discussion 😅
